addons/website_event_track/views/event_track_templates_page.xml

```diff
@@ -128,7 +132,15 @@
                 <div t-field="track.partner_biography" class="oe_no_empty"/>
                 <hr t-if="not is_html_empty(track.description)" class="mt-2 pb-1 mb-1"/>
             </div>
-            <div t-field="track.description" class="my-2 mx-3 oe_no_empty"/>
+
+            <!-- Description -->
+            <t>
+                <div t-if="not is_html_empty(track.description)" t-field="track.description" class="my-2 mx-3 oe_no_empty"/>
+                <div t-elif="not env.user.share" class="my-2 mx-3 oe_no_empty">
```

Comment:
Actually I think we should ensure can effectively edit tracks. We could probably simply check he is event officer (having 'event.group_user'). Otherwise internal users will not be able to add a description as they have no rights to edit tracks.